pkg/koyeb/context.go

```diff
@@ -0,0 +1,44 @@
+package koyeb
+
+import (
+	"context"
+
+	"github.com/koyeb/koyeb-api-client-go/api/v1/koyeb"
+	"github.com/koyeb/koyeb-cli/pkg/koyeb/idmapper"
+	"github.com/koyeb/koyeb-cli/pkg/koyeb/renderer"
+	"github.com/spf13/cobra"
+)
+
+// SetupCLIContext is called by the root command to setup the context for all subcommands.
+func SetupCLIContext(cmd *cobra.Command) {
+	apiClient := getApiClient()
+	ctx := cmd.Context()
+	ctx = context.WithValue(ctx, koyeb.ContextAccessToken, token)
+	ctx = context.WithValue(ctx, "client", apiClient)
+	ctx = context.WithValue(ctx, "mapper", idmapper.NewMapper(ctx, apiClient))
+	ctx = context.WithValue(ctx, "renderer", renderer.NewRenderer(outputFormat))
+	cmd.SetContext(ctx)
+}
+
+type CLIContext struct {
+	context  context.Context
+	client   *koyeb.APIClient
+	mapper   *idmapper.Mapper
+	token    string
+	renderer renderer.Renderer
+}
```

Comment:
Why not making all or some of these fields public? Do you expect this object to only be used inside this package? Then maybe it might be private as well.

```suggestion
type cliContext struct {
	context  context.Context
	client   *koyeb.APIClient
	mapper   *idmapper.Mapper
	token    string
	renderer renderer.Renderer
}
```

Reply:
To be clear, I'd make the fields public, or would define a getter, to accommodate for the possibility of having to use this type outside this package.